core/templates/css/oppia.css

```diff
@@ -4445,3 +4449,285 @@ div.oppia-issues-learner-action {
 .forced-modal-stack {
   z-index: 1035 !important;
 }
+
+.oppia-profile-h1 {
+  color: #fff;
+  display: inline;
```

Comment:
Why we have this CSS here and not on the profile page?

Reply:
These styles were previously present in `profile-page.mainpage.html` file but now this file does not exist. So, I moved these styles to `oppia.css`